src/api_manager/config.cc

```diff
@@ -403,6 +403,17 @@ bool Config::LoadBackends(ApiManagerEnvInterface *env) {
   return true;
 }
 
+bool Config::LoadTypes(ApiManagerEnvInterface *env) {
+  for (const auto &type : service_.types()) {
+    for (const auto &field : type.fields()) {
+      if (field.name().find("_") != std::string::npos) {
+        snake_path_to_json_map_.emplace(field.name(), field.json_name());
+      }
+    }
+  }
+  return true;
```

Comment:
it seems that you always return true,   how use void return type

Reply:
Done.